components/automate-ui/src/app/components/time/time.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+import { UserPreferencesService } from 'app/services/user-preferences/user-preferences.service';
+
+@Component({
+  selector: 'chef-time',
```

Comment:
For components, we use ```app``` as a selector, not a chef. chef is for stencil files.
@vinay033  let me know your opinion on this.

Reply:
@himanshi-chhabra thanks for this correction, we are using `app`. 
@Venkatesh-rengasamy please test this with `app`, you can also check our existing components for reference.

Reply:
done